### DISCUSSION TITLE
Display buttons not quiet and locked.

### DISCUSSION THREAD

**Discussion by gorper** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#557] [General]
Greetings and thanks for an interesting and good project.
Unfortunattly I have some small findings I need to ask about. I have a Vegas with 6 wired 54122 egg shaped pump. I run currently Beta: [2023-06-19-2324] on a pcb v2B, also red lcc and at the moment capacitors. My resistors has not arrived yet.

It work just out of the box as pre 2021 but.
All buttons on display is in correct place, indicator led and segment display works as it should

Unlocked (only short press needed for unlock)
Timer button works, chirps
Bubble button works, chirps
Celcius button works, chirps
Heater button works, chirps
Pump button works, chirps
Temperature buttons ok funny sound

Locked (only short press needed for lock)
Timer button beeps but nothing happens
Bubble button dead
Celcius button dead
Heater button dead
Pump button dead
Temperature button beeps but no action
Power button works and shut off everything.

Timer and temperature buttons beep when shutted off on power button.

As said I am pleased over the work done and just noticed this oddity’s, in the documents it seems the Vegas is not fully known and seem to exist with 4 and 6 wires. I also have tested other pump models but then buttons is misplaced and best operation is reached with the above setting.

**Comment by visualapproach** [comment]
Hello, yes I have on my todo list to disable sound when the pump is locked. So it seems it works as expected and you may as well save the resistors instead of resoldering everything 🙂 
Thanks for your report

**Reply by gorper** [reply]
I read something about the beeps, but got the impression that it was sorted in beta release I refer to. Pardon for that, so only question is then if it is as intended that it is possible to power off from display when pump is locked.

**Reply by visualapproach** [reply]
It's as intended. Actually it's as from the factory I believe. If using the WEB UI it's Auto Unlock but pressing the buttons more or less just passes the messages to the pump.

**Reply by gorper** [reply]
Thanks for reply, but I was still annoyed how it seems to behave for me, so I disconnected project and verified how it works. And yes You are correct, but the differance and what I feel as an issue is the fact that a long press like 3 sec is needed to shutdown when pump is in original shape. When reconnect the project just a quick tap is needed on power button and pump shut down.

**Reply by visualapproach** [reply]
Yes that's correct. What I mean is it is working as intended so your setup is fine, hardware wise. So I take your opinion as a feature request for long press?

**Reply by gorper** [reply]
Thanks for Your interest in my small issue. Yes a feature request absolutely. Because it is a little to easy to unintentionally shut off the tub now. 

The feature request is: Introduce long press 3 sec on lock/unlock button and on power button in locked mode.

**Comment by gorper** [comment]
If the long press to unlock is introduced, it mimic original behaviour better in my opinion.
